data/syndicate jobs.txt

```diff
@@ -306,3 +306,183 @@ ship "Vanguard" "Vanguard Test Dummy"
 	add attributes
 		"automaton" 1
 		"self destruct" 1
+
+
+mission "Syndicate Prisoner Transport [0]"
+	name "Transport indicted executive to house arrest"
```

Comment:
![image](https://user-images.githubusercontent.com/17688683/50331003-2e1c7580-04cb-11e9-83f3-9cb8d1cc13ae.png)
You're going to want to double check the name lengths. Names should be at most like 40 characters. If the name has a `<planet>` text replacement, use `Carbuncle Station` as a worst case scenario.

Reply:
Cut out more cruft in 0151e5c.